src/components/main/recommend-list.tsx

```diff
@@ -28,7 +28,7 @@ const RecommendList: NextPage<RecommendListProps> = ({
 
   return (
     <>
-      {recommendList && recommendList.length === 0 && (
+      {recommendList?.length === 0 ? (
```

Comment:
```suggestion
      const isRecommendList = recommendList?.legnth > 0;
      
      {recommendList?.length === 0 ? (
```
이런식으로 변경하면 삼항연산자에서 어떤 조건에 어떤게 return된다는 것을 명확하게 알 수 있을 것 같습니다.